src/pathfinding_service/model/channel_view.py

```diff
@@ -16,6 +16,26 @@
 from raiden_libs.marshmallow import ChecksumAddress
 
 
+@dataclass
+class FeeSchedule:
+    flat: FeeAmount
+    proportional: float
+    imbalance_penalty: Optional[List[List[FeeAmount]]] = None
```

Comment:
I don't understand this type. Based on what happens below, shouldn't this be `Optional[List[Tuple[TokenAmount, FeeAmount]]]`, a list of pairs `(balance, penalty)`?

Reply:
marshmallow2 has problems with tuples. I'll update this after updating Raiden.